package.json

```diff
@@ -29,10 +29,14 @@
   },
   "dependencies": {
     "autoprefixer": "^10.4.17",
+    "expo-document-picker": "^11.10.1",
+    "expo-image-picker": "~14.3.2",
     "firebase": "^10.7.1",
     "postcss": "^8.4.33",
     "react-bootstrap": "^2.10.0",
     "react-firebase-hooks": "^5.1.1",
+    "react-native-document-picker": "^9.1.1",
+    "react-native-share": "^10.0.2",
```

Comment:
Are all of these dependencies really used or some can be removed?

Reply:
You're right. Didn't need them